pkg/controller/volume/cache/desired_state_of_world_test.go

```diff
@@ -89,7 +89,7 @@ func Test_AddPod_Positive_NewPodNodeExistsVolumeDoesntExist(t *testing.T) {
 	// Arrange
 	volumePluginMgr, _ := volumetesting.GetTestVolumePluginMgr(t)
 	dsw := NewDesiredStateOfWorld(volumePluginMgr)
-	podName := types.UniquePodName("pod-uid")
+	podName := "pod-uid"
```

Comment:
Since you are creating a pod object now. Create that first and then do:
`podName := volumehelper.GetUniquePodName(pod)`